app/graphql/types/location_type.rb

```diff
@@ -13,8 +13,8 @@ class Types::LocationType < Types::BaseObject
   field "moon_phase", Float
   field "solar_cycles", [Types::SolarcycleType]
 
-  field "wifi_name", String
-  field "wifi_password", String
-  field "bathroom_code", String
+  field "wifi_name", String, null: true
+  field "wifi_password", String, null: true
+  field "bathroom_code", String, null: true
```

Comment:
Why would we want to add this (null: true)?

Reply:
I imagine because not all of the locations have wifi or secured bathroom doors

Reply:
@fallynlogan the field  may return nil when it is set to null:true.